cmd/wails/6_serve.go

```diff
@@ -10,16 +11,22 @@ import (
 
 func init() {
 
+	fs := cmd.NewFSHelper()
+
 	var forceRebuild = false
 	var verbose = false
+	var configPath = fs.Cwd()
+	var outPath = filepath.Join(fs.Cwd(), "build")
 	buildSpinner := spinner.NewSpinner()
 	buildSpinner.SetSpinSpeed(50)
 
 	commandDescription := `This command builds then serves your application in bridge mode. Useful for developing your app in a browser.`
 	initCmd := app.Command("serve", "Run your Wails project in bridge mode").
 		LongDescription(commandDescription).
 		BoolFlag("verbose", "Verbose output", &verbose).
-		BoolFlag("f", "Force rebuild of application components", &forceRebuild)
+		BoolFlag("f", "Force rebuild of application components", &forceRebuild).
+		StringFlag("c", "Specify location of project.json", &configPath).
+		StringFlag("o", "Specify where the built executable should be placed", &outPath)
```

Comment:
Is this asking for a directory or a /path/to/filename?

Reply:
`-o` is ultimately passed to `go build`, and the way that works is if you pass a directory it just places there the built executable, if you also pass a filename it also renames the executable. So can be either. `-c` should be a path, not dir